src/shared/utils/price-conversion.ts

```diff
@@ -22,3 +23,25 @@ export const calculateDisplayPrice = (
     return baseCalculatedPrice / divisor;
   }
 };
+
+/**
+ * Calculates USDC-normalized amounts for trading pair values (volume or liquidity)
+ */
+export function calculateEquivalentInUSDC(
+  liquidityOrVolume: number,
+  usdcPrice: number,
+  quoteAssetMetadata: Metadata,
+  usdcMetadata: Metadata,
+): ValueView {
+  const expDiff = Math.abs(
+    getDisplayDenomExponent(quoteAssetMetadata) - getDisplayDenomExponent(usdcMetadata),
+  );
+
+  // TODO: create `pnum.multiply()` utility: https://github.com/penumbra-zone/dex-explorer/issues/231
+  const result = liquidityOrVolume * usdcPrice * 10 ** expDiff;
```

Comment:
suggestion: I see @JasonMHasperhoven using `.times()` function on BigNumber in collab with `pnum`. Maybe this could help? https://github.com/penumbra-zone/dex-explorer/blob/main/src/pages/trade/model/positions.ts#L285